common/web/input-processor/src/text/prediction/languageProcessor.ts

```diff
@@ -214,7 +214,7 @@ export default class LanguageProcessor extends EventEmitter<LanguageProcessorEve
     // This may refer to the context before an input keystroke or before application
     // of a predictive suggestion.
     let original = this.getPredictionState(suggestion.transformId);
-    if(!original) {
+    if(!original || !this.isConfigured) {
```

Comment:
I wonder if we should have separate warnings that are logged as events to sentry? After all, why are either of these things happening?